projects/keycloak/SamlParserFuzzer.java

```diff
@@ -79,7 +339,9 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) {
       ByteArrayInputStream bais = new ByteArrayInputStream(input);
       XMLEventReader reader = XMLInputFactory.newInstance().createXMLEventReader(bais);
 
-      parser.parse(reader);
+      if (parser != null) {
```

Comment:
Why do we want to make this check?

Reply:
Just for fail safe because I don't have default label in the switch. Although it should not happen, but it is better to have the fail safe case.

Reply:
Why is it better to have the fail safe case here?

Reply:
It shouldn't happen that it is `null` here, and if it does, we want to know IMO

Reply:
Ok. I will fix it.